modules/javafx.controls/src/main/java/javafx/scene/control/Cell.java

```diff
@@ -596,6 +596,7 @@ public void cancelEdit() {
     }
 
     /**
+     * Commits an edit to the value of the cell.
```

Comment:
A similar change could be made to `startEdit` and `cancelEdit` in the same class.